tools/Azure.Mcp.Tools.AzureBestPractices/src/Resources/azure-general-deployment-best-practices.txt

```diff
@@ -19,6 +22,9 @@ Quality requirements for IaC files:
 - Clear organization
 - Separation of concerns
 - **Always** place Bicep files under the 'infra/' folder unless the user specifies a different location
+- When configuring infra-as-code for Azure Key Vault, DO NOT disable purge protection.
+- When configuring infra-as-code for Container Registry, DO NOT enable anonymous pull access.
+- When configuring infra-as-code for Azure resources, ALWAYS use the latest API version available.
```

Comment:
This will be very hard for the model to follow exactly. I think practically what we should expect the model to do is to use the bicepschema tool to get the schema and use whatever schema the tool returns and passes the MCP host validation (e.g. Bicep language service).